coil-sample-compose/src/main/java/sample/compose/LazyStaggeredGrid.kt

```diff
@@ -49,7 +50,7 @@ fun LazyStaggeredGrid(
     }
     val gridScope = RealLazyStaggeredGridScope(columnCount).apply(content)
 
-    Row(Modifier.scrollable(scrollableState, Orientation.Vertical)) {
+    Row(Modifier.scrollable(scrollableState, Orientation.Vertical).testTag("scrollableRow")) {
```

Comment:
Can we pass a custom `Modifier` to the function? I'd like to keep `LazyStaggeredGrid` pretty generic so it can be copy/pasted into other projects.

Reply:
Done